File: SandBox/Ivona/fronend/my_app/lib/series.dart
class Series{
  int id;
  String name;
  String genre;
  String theDirector;
  int beginYear;
  int endYear;

  Series(Name, Genre, TheDirector, BeginYear, EndYear){
    this.name = Name;
    this.genre = Genre;
    this.theDirector = TheDirector;
    this.beginYear = BeginYear;
    this.endYear = EndYear;
  }

  Series.WithID(MovieID, Name, Genre, TheDirector, BeginYear, EndYear){
    this.id = MovieID;
    this.name = Name;
    this.genre = Genre;
    this.theDirector = TheDirector;
    this.beginYear = BeginYear;
    this.endYear = EndYear;
  }

  Map<String,dynamic> toMap(){
    var map = Map<String,dynamic>();
    map['ID'] = id;
    map['Name'] = name;
    map['Genre'] = genre;
    map['TheDirector'] = theDirector;
    map['BeginYear'] = beginYear;
    map['EndYear'] = endYear;
    return map;
  }


}

File: SandBox/Ivona/backend/demoApp/Models/Series.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace demoApp.Models
{
    public class Series
    {
        public int ID { get; set; }
        public string Name { get; set; }
        public string Genre { get; set; }
        public string TheDirector { get; set; }
        public int BeginYear { get; set; }
        public int EndYear { get; set; }
    }
}
